### PR TITLE
[NEW] Nickname and Bio in profileView

### PR DESCRIPTION
**Pull request by EzequielDeOliveira**
## Proposed changes
Fields to Nickname and Bio added to profileView like the profileView of web client

#### Web client 
image

#### Mobile 
image

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/views/ProfileView/styles.js [hunk 1]
```diff
@@ -23,5 +23,10 @@ export default StyleSheet.create({
 		marginRight: 15,
 		marginBottom: 15,
 		borderRadius: 2
+	},
+	bio: {
+		height: '5%',
```
Comment (diegolmello) [inline app/views/ProfileView/styles.js:28]: Don't use fixed heights for this.
`minHeight` is more suitable.

File: app/views/ProfileView/index.js [hunk 2]
```diff
@@ -361,6 +378,73 @@ class ProfileView extends React.Component {
...
+
+		return (
+			<RCTextInput
+				editable
+				inputStyle={[
+					!true && styles.disabled,
+					styles.bio
+				]}
+				numberOfLines={2}
+				maxLength={260}
```
Comment (diegolmello) [inline app/views/ProfileView/index.js:434]: Where did you get this from?

File: app/views/ProfileView/index.js [hunk 3]
```diff
@@ -361,6 +378,73 @@ class ProfileView extends React.Component {
...
+	renderBioField = () => {
+		const {
+			bio
+		} = this.state;
+		const {
+			theme, serverVersion
+		} = this.props;
+
+		// Bio was added to release 3.5.0
+		if (semver.lte(serverVersion, '3.1.0')) {
```
Comment (diegolmello) [inline app/views/ProfileView/index.js:422]: Which one is correct?

File: app/views/ProfileView/index.js [hunk 4]
```diff
@@ -361,6 +378,73 @@ class ProfileView extends React.Component {
...
+			return null;
+		}
+
+		return (
+			<RCTextInput
+				editable
+				inputStyle={[
+					!true && styles.disabled
+				]}
+				maxLength={120}
```
Comment (diegolmello) [inline app/views/ProfileView/index.js:400]: Where did you get this from?

File: app/i18n/locales/en.js [hunk 5]
```diff
@@ -114,6 +114,7 @@ export default {
 	Avatar_Url: 'Avatar URL',
 	Away: 'Away',
 	Back: 'Back',
+	Bio: 'Bio',
```
Comment (diegolmello) [inline app/i18n/locales/en.js:117]: Missing pt-BR

File: app/views/ProfileView/index.js (hunk 2 above)
Comment (EzequielDeOliveira) [reply app/views/ProfileView/index.js:429]: https://github.com/RocketChat/Rocket.Chat/blob/10ea922d89b05f358c656996913d6492edd4c417/server/methods/saveUserProfile.js#L50

File: app/views/ProfileView/index.js (hunk 4 above)
Comment (EzequielDeOliveira) [reply app/views/ProfileView/index.js:397]: https://github.com/RocketChat/Rocket.Chat/blob/10ea922d89b05f358c656996913d6492edd4c417/server/methods/saveUserProfile.js#L58

Comment (EzequielDeOliveira) [reply app/views/ProfileView/index.js:397]: Should I put this link as a comment?

Comment (diegolmello) [reply app/views/ProfileView/index.js:397]: 😱  That's so ugly!
Yeah, please add the line linking to the server repo.
Thanks.

Comment (EzequielDeOliveira) [reply app/views/ProfileView/index.js:397]: Ok, Thanks!

**Comment by diegolmello** [comment]
We've been talking to design team about where/how we should apply nickname on the rest of the app.
I'll finish this PR when we agree on that.
Thanks! :)

**Pull request closed without merging**